mantle/kola/tests/misc/network.go

```diff
@@ -56,6 +56,15 @@ func init() {
 		Distros:     []string{"rhcos"},
 		Platforms:   []string{"qemu-unpriv"},
 	})
+	// This test follows the same network configuration used on https://github.com/RHsyseng/rhcos-slb
+	// with a slight change, where the MCO script is run from ignition: https://github.com/RHsyseng/rhcos-slb/blob/main/setup-ovs.sh.
+	// and we're using veth pairs instead of real nic when setting the bond
+	register.RegisterTest(&register.Test{
+		Run:         NetworkBondWithDhcp,
+		ClusterSize: 0,
+		Name:        "rhcos.network.bondWithDhcp",
```

Comment:
Minor/optional: our naming scheme for tests is usually kebab-casing (see `kola list`).

Reply:
np. Done